src/test/java/io/xstefank/wildfly/bot/PRDirectoriesVerificationTest.java

```diff
@@ -0,0 +1,180 @@
+package io.xstefank.wildfly.bot;
+
+import io.quarkiverse.githubapp.testing.GitHubAppTest;
+import io.quarkus.test.junit.QuarkusTest;
+import io.xstefank.wildfly.bot.model.RuntimeConstants;
+import io.xstefank.wildfly.bot.utils.GitHubJson;
+import io.xstefank.wildfly.bot.utils.MockedGHPullRequestProcessor;
+import io.xstefank.wildfly.bot.utils.TestConstants;
+import io.xstefank.wildfly.bot.utils.Util;
+import org.apache.commons.io.IOUtils;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.kohsuke.github.GHCommitState;
+import org.kohsuke.github.GHContent;
+import org.kohsuke.github.GHEvent;
+import org.kohsuke.github.GHRepository;
+import org.mockito.Mockito;
+
+import java.io.IOException;
+import java.util.Set;
+
+import static io.quarkiverse.githubapp.testing.GitHubAppTesting.given;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@QuarkusTest
+@GitHubAppTest
+public class PRDirectoriesVerificationTest {
+
+    private static GitHubJson gitHubJson;
+    private MockedGHPullRequestProcessor mockedContext;
+
+    @BeforeAll
+    public static void setupTests() throws IOException {
+        gitHubJson = GitHubJson.builder(TestConstants.VALID_PR_TEMPLATE_JSON)
```

Comment:
this can be in the field definition

Reply:
I do not think so, IIRC @petrberan tried it and it won't work due to file loading